src/browser/provider/built-in/base.js

```diff
@@ -15,10 +16,22 @@ function getCurrentSize () {
         height: window.innerHeight,
     };
 }
+
+function getMacSizeInfo () {
```

Comment:
`getWindowDimensionsInfo` - it's not related to Mac on it's own, so we can ommit `Mac` from the name